futile.lua

```diff
@@ -0,0 +1,32 @@
+isPhoneOpen = false
```

Comment:
You don’t need to define the variable

Reply:
Actually you do because of `isPhoneOpen = not isPhoneOpen`

You could of just done the whole if true or false stuff but your way seems cleaner. 
I would of passed the arg to the event to denote if it’s enable or disabled for easier refactoring but to each their own.